packages/mobile/views/dashboard/DashboardRouter.svelte

```diff
@@ -32,6 +41,11 @@
         $selectedActivity = null
         $dashboardRouter.previous()
     }
+    function onTokenDrawerClose(): void {
+        resetRouterWithDrawerDelay($tokenRouter)
+        $selectedAsset = null
+        $dashboardRouter.previous()
```

Comment:
on send I implemented the function `closeDrawer` which centralizes the logic of closing the drawer, might be interesting also here

Reply:
I just didnt refactor all the routes to refactor them little by little